tests/collections/nlp/test_nlp_exportables.py

```diff
@@ -103,6 +104,28 @@ def test_IntentSlotClassificationModel(self, dummy_data):
             assert onnx_model.graph.output[0].name == 'intent_logits'
             assert onnx_model.graph.output[1].name == 'slot_logits'
 
+    @pytest.mark.run_only_on('GPU')
+    @pytest.mark.unit
+    def test_IntentSlotClassificationModel_export_to_torchscript(self, dummy_data):
```

Comment:
This method needs to be extended by some basic assertions like the one below, `test_TokenClassificationModel_export_to_onnx`

Reply:
Added some of those. Though torchscript model internals seem to be a bit more difficult to inspect -- hopefully the assertions make sense.